.circleci/config.yml

```diff
@@ -333,3 +374,12 @@ workflows:
             - build-docker-image
             - test-erlang-22_elixir-1-9
           filters: *all_tags
+
+      - gh-pages-deploy:
+          requires:
+            - pre-build
+          filters:
+            tags:
+              only: /^\d+\.\d+\.\d+([a-z0-9\-\+\.])*/
+            branches:
+              only: cci-doc
```

Comment:
Another change to the `master` branch. I wonder if we want to do it in the PR or after it's merged.

Reply:
when the PR is good to merge, then I'll revert this temp change for testing